app/Http/Livewire/EditCourseModal.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace App\Http\Livewire;
+
+use App\Concerns\HasEditModal;
+use App\Models\Course;
+use App\Types\CourseType;
+use Illuminate\Support\Str;
+use Livewire\Component;
+
+class EditCourseModal extends Component
+{
+    use HasEditModal;
+
+    protected $listeners = ['show'];
+    public $modalName;
+    public $showModal = false;
+
+    protected $course;
+
+    public function mount($errorBag = null)
+    {
+        if ($errorBag != null) {
+            $this->setErrorBag($errorBag);
+        }
+
+        $this->modalName = Str::kebab(class_basename($this));
+
+        if (! $errorBag->isEmpty()) {
```

Comment:
This might lead to a Runtime Error when `$errorBag == null`.

Reply:
Yes it would. @gautamswati  Please make changes as I suggested.

Reply:
yeah, okay.